src/pydata_sphinx_theme/assets/scripts/pydata-sphinx-theme.js

```diff
@@ -717,16 +743,21 @@ function debounce(callback, wait) {
     }, wait);
   };
 }
+// Determining whether an element has scrollable content depends on stylesheets,
+// so we're checking for the "load" event rather than "DOMContentLoaded"
+if (document.readyState === "complete") {
+  addTabStopsToScrollableElements();
+} else {
```

Comment:
I had to add the `document.readyState` check in order for the Playwright tests to pass. It may or may not be necessary in other browsers, but it makes sense to keep it as a bit of defensive programming.